index.html

```diff
@@ -15,24 +15,52 @@
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+    <header>
+        <nav>
+            <a href="#">Awesome Books</a>
+            <ul class="navLinkCont">
+                <li><button type="button" class="navLink" id="list">List</button></li>
+                <li><button type="button" class="navLink" id="addNew">Add new</button></li>
+                <li><button type="button" class="navLink" id="contact">Contact</button></li>
+            </ul>
+        </nav>
+        <span class="date">Hello</span>
+    </header>
     <main>
-        <h1>All awesome Books</h1>
-        <form>
+        <section id="bookList" class="hidden">
+        <h1 class="bookHeader">All awesome Books</h1>
             <table id="bookContainer">
             <!-- <label id="title"></label><br>
             <label id="author"></label><br>
             <input class="remove" type="button" value="Remove">
             <hr> -->
         </table>
-        <div class="formCont">
-            <hr class ="separator">
-            <h2 class="addBook">Add a new book</h2>
+    </section>
+        <form>
+        <div class="formCont hidden">
+            <h2 class="subheading">Add a new book</h2>
             <input class="inputField" id="titleInput" type="text" placeholder="Title"><br>
```

Comment:
- [ OPTIONAL ] Consider adding this feature - After the student adds a new book the inputs can disappear, this would be great for user experience

Reply:
Thank you @GzimAsani for this great idea! We have taken your feedback into consideration and now the input field clears after a book has been added.